### PR TITLE
Microbit landing page

### PR DESCRIPTION
**Pull request by ntoll**
This static page contains links relating to MicroPython running on the BBC micro:bit. In order to comply with BBC rules and UK child protection laws about linking to external sites, all embedded social-media have been removed from the template.

One for @tjguk to review please.

### CODE REVIEW

File: pydotorg/urls.py [hunk 1]
```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```
Comment (berkerpeksag) [inline pydotorg/urls.py:1]: These unnecessary changes (perhaps caused by different editor configuration?) makes code review a little bit harder.

Comment (ntoll) [inline pydotorg/urls.py:1]: These were from @tjguk who did the initial job of working out how this should be structured. The new line 50 of this file is the important thing to retain.

Comment (berkerpeksag) [inline pydotorg/urls.py:1]: I'm not talking about line 50 or structure of the change. Most of the changes in this file are only whitespace changes which I think caused by different editor configuration. See the diff between https://github.com/python/pythondotorg/pull/894/files (with whitespaces) and https://github.com/python/pythondotorg/pull/894/files?w=1 (without whitespaces)

Comment (ntoll) [inline pydotorg/urls.py:1]: I totally get that and understand why it's happened (@tjguk is a Windows user so the line endings are probably different). Just ensuring that _the important change_ wasn't lost in the mega-diff. ;-)

Comment (tjguk) [inline pydotorg/urls.py:1]: As @ntoll says this is likely down to my Windows setup. (I assume it's
because he's worked from a zipped patch I sent over which would have
used my line-endings: my gitconfig has suitable eol configs set).

@ntoll, if you have the time to run a converter over the PR to smooth
out the eol diffs, feel free. Otherwise I'm happy to get to it myself
but it won't be today.

Comment (ntoll) [inline pydotorg/urls.py:1]: On it now. :-)

And, as always, many many thanks @tjguk.

Comment (ntoll) [inline pydotorg/urls.py:1]: _sigh_ So GIT is telling me there is no difference except for line 50 between this branch and master. It also says that everything is up-to-date. However, GitHub says something different.

I give up. :-(

@tjguk do you have an ETA for when this could be fixed. I have the BBC breathing down my neck.

Comment (berkerpeksag) [inline pydotorg/urls.py:1]: I can fix it today :)

Comment (tjguk) [inline pydotorg/urls.py:1]: On 29/02/2016 08:56, Berker Peksag wrote:

> In pydotorg/urls.py
> https://github.com/python/pythondotorg/pull/894#discussion_r54382722:
> 
> > @@ -1,71 +1,72 @@
> > -from django.conf.urls import include, url, handler404
> 
> I can fix it today :)

Please go ahead, Berker. I certainly won't get to it until late tonight
and most likely tomorrow.

Also, I had to guess somewhat as to the correct approach; the idea is to
have a page which fits the broad python.org style but without the
Twitter &c. trappings from the standard template as this contravene's
the BBC's guidelines.

Comment (berkerpeksag) [inline pydotorg/urls.py:1]: Done.

Do we need to remove the "Socialize" menu from the header?

Comment (ntoll) [inline pydotorg/urls.py:1]: Nope... it's good as it stands. The problem was that the page not contain embedded social media. There's nothing to stop us linking to it. What we have is compliant with the BBC rules.

Comment (berkerpeksag) [inline pydotorg/urls.py:1]: Great, here's my branch: https://github.com/python/pythondotorg/compare/master...berkerpeksag:pr/894 If it looks good to both of you, I can merge and deploy it.

Comment (ntoll) [inline pydotorg/urls.py:1]: Looks good to me! Many many thanks for your help @berkerpeksag. Looking forward to seeing this deployed on python.org :-)

**Comment by berkerpeksag** [comment]
Merged in https://github.com/python/pythondotorg/commit/cdbb5be8d92e145a30dfdc3b50ecb1bde2fc5256 and deployed. Staging version is already up at https://staging.python.org/community/microbit/ Thanks! :)

**Comment by tjguk** [comment]
Thanks for sorting out the mess I left! It looks good.

**Comment by ntoll** [comment]
When will it arrive in the live website..? Just asking so I can manage upstream (BBC) expectations of timings.

**Comment by berkerpeksag** [comment]
It's already online for me: https://www.python.org/community/microbit/ Please let me know if you still can't see it on python.org.

**Comment by tjguk** [comment]
I originally got a 503 with some kind of Varnish error but I refreshed
and it's there now.

**Comment by ntoll** [comment]
Yup.... I can see it. Many thanks to you both for your help! :-)
